lunar/distribution.yaml

```diff
@@ -710,6 +710,22 @@ repositories:
       url: https://github.com/orocos/orocos_kinematics_dynamics.git
       version: master
     status: maintained
+  pcl_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/pcl_msgs.git
+      version: indigo-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/pcl_msgs-release.git
+      version: 0.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
I was able to add @ros-pull-request-builder to the org unit instead. That way you don't have to add the user to each repo and don't have to create the webhooks manually.